### PR TITLE
Portable chem mixer: Integration of fermichem and minor updates to usability

### PR DESCRIPTION
**Pull request by LordVollkorn**
## About The Pull Request

This PR consists of the following changes:
- The portable chem mixer now displays pH-values
- It can now dispense 1u of reagents, so buffer solutions can be added carefully
- It can now be loaded with bottles, plastic bottles and condiment bottles (all bottle types that the chem masters/chemistry can make). Beakers also still work of course.
- CTRL+click opens and closes it to switch between filling mode (inventory in which you place beakers and bottles) and dispensing mode (UI in which you dispense to a container)
- When worn on the belt and in filling mode, a left click opens its inventory. No more dragging or alt-clicking needed to open it. (alt-clicking to swap beakers in dispense mode still works, just like the normal dispensers)
- The UI has been resized to, in its default state, make it easier to visually skim through the chemicals in the list

UIchange

bottles

## Why It's Good For The Game

- Better integrates with the new pH system
- Can now actually load the types of bottles that botany puts their chemicals in, as well as the water bottles chemists usually make
- Less clunky/frustrating to use due to the new click/key+click functions
- Better overview for faster selections

Personal note: Its a good storage device for pure precursor chemicals that are needed for different reactions later on, e.g. keeping 100u of highly pure phenol to use when making salicylic acid or oxandrolone.
Also keeps the chemical fridges free for actual medicine and the floors free of dropped bottles and beakers (in theory).

## Changelog
:cl:
code: The Portable Chemical Mixer has been improved. Don't know where to efficiently put all your bottles of pure precursor chemicals and medicines? Fridges and floors littered with half-filled beakers? Just print the portable chemical mixer at the medbay lathe, no anomaly core needed!
tweak: It now displays pH-values.
tweak: It can dispense smaller amounts (1u).
tweak: It can now also be loaded with bottles, plastic bottles and condiment bottles.
tweak: It can be opened and closed with CTRL+left click.
tweak: It's inventory (beakers and bottles) can be accessed with a simple left click now when on the belt (when its opened).
tweak: It's UI has been resized  to make it easier to visually skim through the chemicals in it's list.
/:cl:

### CODE REVIEW

File: tgui/packages/tgui/interfaces/PortableChemMixer.js [hunk 1]
```diff
@@ -42,9 +42,10 @@ export const PortableChemMixer = (props, context) => {
               <Button
                 key={chemical.id}
                 icon="tint"
-                width="150px"
-                lineHeight="21px"
+                width="425px"
```
Comment (stylemistake) [inline tgui/packages/tgui/interfaces/PortableChemMixer.js:45]: use `fluid` instead of width, since you want 100% width anyway

Comment (LordVollkorn) [reply tgui/packages/tgui/interfaces/PortableChemMixer.js:45]: yessir

Comment (stylemistake) [reply tgui/packages/tgui/interfaces/PortableChemMixer.js:45]: and you can remove mr={-1} in the box above too

Comment (LordVollkorn) [reply tgui/packages/tgui/interfaces/PortableChemMixer.js]: Is the change I made alright? I dont know if I commited a crime with my coding

**Review by stylemistake — APPROVED** [review]

**Pull request merged**